probablility.py

```diff
@@ -80,4 +80,4 @@ def printInGrid(numbers):
             print(numbers[i], end="  ")
     print('\n\n')
 
-playNumberOfTimes(100)
+playNumberOfTimes(100000)
```

Comment:
It looks weird to hardcode this magic number.
Wouldn’t it make more sense to read it from command line argument?

`playNumberOfTimes(sys.argv[1])`

https://www.geeksforgeeks.org/command-line-arguments-in-python/amp/